package/functions/Funcs/misc/packageName.js

```diff
@@ -0,0 +1,11 @@
+const { name } = require('../../../../package.json');
+
+module.exports = d => {
+    let { code, result } = d.util.openFunc(d);
+
+    result = name;
+
+    return {
+        code: d.util.setCode({ function: d.func, code, result })
+    }
+}
```

Comment:
this is deprecated

Reply:
So do i remove it?

Reply:
I just added according to the changelog of v4.5.0

Reply:
> So do i remove it?

yes remove it @dbthesecond u can see its -$packageName in changelog